src/main.ts

```diff
@@ -48,7 +49,13 @@ function actionHandler(file: Node, dir: string) {
       path: dir,
     });
   } else {
-    downloadUrl = getAbsolutePath(file.encodedSource);
+	// NC28 dependency : Node.encodedSource
```

Comment:
Could you rephrase the comment to instead say that pre-NC28 doesn't have Node.encodedSource?
Here it makes it sound like `Node.encodedSource` is a dependency of NC28 and thus required by NC28 (it's not) which is confusing.